components/fileqcs/fileQcsController.js

```diff
@@ -340,11 +341,20 @@ function getIndexedPlaceholders (projectNames) {
   return projectNames.map((item, index) => '$' + (index + 1)).join(', ');
 }
 
+function getQuotedPlaceholders (workflowNames) {
+  return workflowNames.split(',')
```

Comment:
Why not `join(', ')`?

Reply:
The string will be in the format `CASAVA,BWA,GATK` and I need it in the format `'CASAVA','BWA','GATK'`.